website/content/users/qsim.md

```diff
@@ -0,0 +1,592 @@
+
+All workflows accept an optional `evaluator` key in addition to their set of configuration parameters to specify the `CostFunctionEvaluator` (via a name string or an instantiated instance). If none provided, the partial-tomography (`default`) one will be used.
+
+The `CostFunctionEvaluator` configuration parameters, if any, can be provided in the workflow parameter pack, which will be forwarded to the `CostFunctionEvaluator` appropriately.
+
+The `CostFunctionEvaluator` will use the runtime QPU instance to execute the necessary sub-circuits to evaluate the operator expectation value. 
+
+### <a id="partial-tomo"></a> Partial Tomography 
+
+The `default` CostFunctionEvaluator is based on the partial tomography method, whereby change-of-basis gates are added to measure the expectation values of each product term in the operator.
+
+Users don't need to provide the `evaluator` key (and set it to `default`) when creating the workflow if they wish to use this method because all workflows assume this option.
+
+It's worth noting that error mitigation (if any) for this cost evaluator is performed at the QPU level via conventional QCOR error mitigation decorators. 
+
+### <a id="phase-est"></a> Quantum Phase Estimation
+
+The `qpe` cost evaluator is based on the time-series quantum phase estimation method to estimate the expectation value. In particular, it makes use of the [Prony](https://en.wikipedia.org/wiki/Prony%27s_method) signal processing method to extract the expectation value estimate from the time-series QPE signal.
+
+The `qpe` cost evaluator has the following configuration parameters:
+
+|   Param Key        |                  Description                                     | 
+|--------------------|------------------------------------------------------------------|
+| `steps`            | The number of data points for classical signal processing. Default is 5, which is the minimum number of samples to estimate the energy of two-eigenvalue operators.|
+| `verified`            | (True/False) If true, it will run the verified phase estimation protocol as described in this [paper](https://arxiv.org/pdf/2010.02538.pdf).|
+
+The `qpe` cost evaluator can be explicitly instantiated by using the `getObjEvaluator` helper function and providing the
+operator to be evaluated, the name key (`qpe`), and any additional parameters.
+
+For example,
+
+- In C++:
+
+```cpp
+auto observable = Z(0) + Z(1) + Z(2);
+auto vqpeEvaluator = qsim::getObjEvaluator(observable, "qpe", {{"verified", true}});
+```
+
+- In Python:
+
+```python
+observable = Z(0) + Z(1) + Z(2)
+vqpeEvaluator = qsim.getObjEvaluator(observable, 'qpe', {'verified': True})
+```
+
+## <a id="new-workflow"></a> Implement a new workflow
```

Comment:
Can you move this section to its own page in the Developer Guide? I think it would still be useful to keep this section heading here but just have the text below it say head over to this link in the developer guide for creating a new workflow.